labextension/package.json

```diff
@@ -40,18 +40,21 @@
     "watch": "run-p watch:src watch:labextension",
     "watch:src": "jlpm build:lib -w --preserveWatchOutput",
     "watch:labextension": "jupyter labextension watch .",
-    "deduplicate": "yarn-deduplicate -s fewer --fail"
+    "deduplicate": "yarn-deduplicate -s fewer --fail",
+    "lint:prettier": "prettier --write \"../.github/**/*.yaml\" \"../*.{yaml,yml,md}\" \"../docs/**/*.md\" \"src/**/*.{tsx,ts}\" \"./*.{yml,json,md}\""
   },
   "dependencies": {
-    "@jupyterlab/application": "^2.0 || ^3.0 || ^4.0",
-    "@jupyterlab/launcher": "^2.0 || ^3.0 || ^4.0"
+    "@jupyterlab/application": "^3.0 || ^4.0",
+    "@jupyterlab/filebrowser": "^3.0 || ^4.0",
```

Comment:
We have stopped testing jupyterlab 2 according to 4.0.0 changelog, and this is now also removing support for it right?

Reply:
Ah, nevermind, we said:

> jupyter-server-proxy's JupyterLab extension dropped support for JupyterLab 2 (https://github.com/jupyterhub/jupyter-server-proxy/pull/394)

Then we have some cleanup in other places about this to do.